__tests__/wallet/partialTxProposal.test.ts

```diff
@@ -0,0 +1,517 @@
+/**
```

Comment:
I really like this unit test file but now that we have integration tests in the lib, I feel we could add integration tests for atomic swap as well, without the mocks.